.coderabbit.yml

```
@@ -0,0 +1,361 @@
+language: "en-US"
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)